mevislab.github.io/content/tutorials/visualization/visualizationexample8.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Example 8: Vessel Segmentation using SoVascularSystem"
+date: 2023-12-08
+status: "OK"
+draft: false
+weight: 592
+tags: ["Advanced", "Tutorial", "Visualization", "3D", "Vessel Segmentation"]
+menu: 
+  main:
+    identifier: "visualization_example8"
+    title: "Vessel Segmentation using SoVascularSystem."
+    weight: 592
+    parent: "visualization"
+---
+# Example 8: Vessel Segmentation using SoVascularSystem {#TutorialVisualizationExample8}
+
+## Introduction
+In this tutorial, we'll walk you through the steps to craft a visually striking portrayal of vascular structures by harnessing the capabilities of the `DtfSkeletonization` module. By combining the vessel centerline extraction power of `DtfSkeletonization` with the immersive 3D visualization provided by the `SoVascularSystem` module, you'll effortlessly explore and interact with the intricacies of the resulting vascular system. This integration enhances your ability to comprehensively understand and analyze complex vascular structures.
```

Comment:
Keep it simple :)
In this tutorial, we are using an input mask to create a vessel centerline using the `DtfSkeletonization` module and visualize the vascular structures in 3D using the `SoVascularSystem` module.